x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/get_threat_list.ts

```diff
@@ -62,6 +65,7 @@ export const getThreatList = async ({
         },
       ],
       search_after: searchAfter,
+      // @ts-expect-error is not compatible with SortCombinations
```

Comment:
Tagging @ecezalp for a quick 👁️

Reply:
LGTM, we have a lot of work coming up in 8.1 in this section, I'll keep an eye on it.